### PR TITLE
Now displays the actual photo in the display fragment.

### PR DESCRIPTION
**Pull request by sammyers**
Also hooked up more buttons.

### CODE REVIEW

File: MoodCookie/app/src/main/java/cecelia/moodcookie/ConfirmationPageFragment.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 public class ConfirmationPageFragment extends Fragment {
 
-    private CameraInterface mInterface;
+    private MainActivity mainActivity;
```
Comment (cbauerswald) [inline MoodCookie/app/src/main/java/cecelia/moodcookie/ConfirmationPageFragment.java:17]: Why did you change this? / Why did it exist in the first place?

**Comment by cbauerswald** [comment]
LGTM

File: MoodCookie/app/src/main/java/cecelia/moodcookie/ConfirmationPageFragment.java (hunk 1 above)
Comment (sammyers) [reply MoodCookie/app/src/main/java/cecelia/moodcookie/ConfirmationPageFragment.java:17]: This was originally an interface that the Main Activity implemented to give fragments a way to invoke its methods. I realized that since we only had one Activity it was easier to just cast the return from getActivity() to a Main Activity type
